cfme/tests/cloud_infra_common/test_html5_vm_console.py

```diff
@@ -70,17 +77,21 @@ def test_html5_vm_console(appliance, provider, configure_websocket, vm_obj,
         - A command that creates a file will be sent through the console.
         - Using ssh we will check that the command worked (i.e. that the file
           was created.
+    In the latest 5.11 build, for VMware, if VNC is not available for a VM, CFME Falls forward to
+    WebMKS. To avoid that from happening, make sure your VM(.vmx)/Template(.vmtx) file
+    has following two lines in it:
+
+    RemoteDisplay.vnc.enabled = "true"
+    RemoteDisplay.vnc.port = "5900"
```

Comment:
Is there anyway to verify this in the test case itself? And skip if that is not the case?

Reply:
I did look into it a bit, its completely inside VMware, not sure we can really access this easily. And we don't need to verify this as I am validating console type to be `VNC` in test case below. So by that assertion if I get webMKS mistakenly, the test would fail and you'd need to go and Fix the template. Once the Console_Template is fixed with above settings, it consistently yields VNC console.  Makes sense?